tests/resources/texts/api/test_cts.py

```diff
@@ -676,8 +676,8 @@ def test_parentId(self):
             "ParentId should resolve"
         )
         self.assertIsInstance(
-            passage.parent, Passage,
-            "Parent Passage should be a passage"
+            passage.parent, CtsPassage,
+            "Parent CapitainsCtsPassage should be a passage"
```

Comment:
Are CtsPassage and CapitainsCtsPassage the same thing? This makes it look like they are.